api/v1beta1/jujucluster_types.go

```diff
@@ -45,7 +183,16 @@ type JujuClusterSpec struct {
 	// juju controller. Should be cluster, loadbalancer, or external
 	//+kubebuilder:default="cluster"
 	ControllerServiceType string `json:"controllerServiceType"`
-	CloudEndpoint         string `json:"cloudEndpoint"`
```

Comment:
cloud endpoint was removed as this is now provided in the cloud portion of the spec